modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/graphql/servlet/GraphQLContributor.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.vulcan.graphql.servlet;
+
+/**
+ * @author Javier de Arcos
+ */
+public interface GraphQLContributor {
+
+	public Object getMutationContribution();
+
```

Comment:
I doubt about creating default methods returning null for getMutationContribution and getQueryContribution so the client only needs to implement the one that he need, and not add another implementation returning null

Reply:
I would do it, it'll be cleaner that way.

Reply:
Great. I'll change it right now